### ISSUE TITLE
Fix UI bug when displaying an ePub

### ISSUE THREAD

**Issue by benoit74** [openzim/kolibri#63]
When displaying an ePub with the ePub.js reader, for some reason the page is moved about 30 pixels up and the top of the page disappear.

A hack has been implemented with https://github.com/openzim/kolibri/commit/c543c9d6b92d1dbcf951fcfaa4dcca37d0458c5b to fix the issue on the short term (ensure that everything is visible) but the bug should be fixed.

This might be tackled together or automatically fixed by #55

**Comment by githyuvi** [comment]
@benoit74  For new ui, have you thought of creating new vue components to display doc, videos etc. Or continue using the HTML CSS generated by kolibri2zim.

**Comment by benoit74** [comment]
Yes, we want to create new vue components, for ePub and PDF it is issue #55.
